src/Administration/Resources/app/administration/src/app/component/data-grid/sw-data-grid-settings/sw-data-grid-settings.html.twig

```diff
@@ -79,11 +79,12 @@
         >
 
             {% block sw_data_grid_settings_column_visibility_checkbox %}
-            <sw-checkbox-field
+
```

Comment:
Please remove these unnecessary new lines 💙

Reply:
Of course 💙